src/tequila/quantumchemistry/orbital_optimizer.py

```diff
@@ -217,8 +262,17 @@ def kernel(self, h1, h2, *args, **kwargs):
                 optimizer_arguments = self.vqe_solver_arguments["optimizer_arguments"]
             if self.silent is not None and "silent" not in optimizer_arguments:
                 optimizer_arguments["silent"] = True
+            
+            if not self.history:
```

Comment:
regards the whole if/else block
`initial_values` will be in `optimizer_arguments` and not in `vqe_solver_arguments`

Reply:
I'm sure what you mean here?